### DISCUSSION TITLE
Cypress Crashed when Running test and Getting Error : Error: read ECONNRESET. Please help

### DISCUSSION THREAD

**Discussion by Pratyushwildnet** [cypress-io/cypress#14872] [Cypress Cloud]
Hi all,

I am just started working on cypress and I installed cypress and Node.js but when I run Examples given by cypress then Cypress getting crashed and getting an error: 

Error: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:205:27)
 {
  errno: 'ECONNRESET',
  code: 'ECONNRESET',
  syscall: 'read'
}

Please anyone help me with this.

OS: window 10
Node js : 12.18.3

Please help me

**Comment by Pratyushwildnet** [comment]
@eventualbuddha @jennifer-shehane Please comment and Help.

**Comment by sangmaijibble** [comment]
https://github.com/cypress-io/cypress/discussions/15667
I get the same issues.

**Comment by felikf** [comment]
Same for me:

```
Error: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:209:20)
 {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}
Error: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:209:20)

```

I guess it has to do something with my proxy settings - I think it started after I've set corporate proxy.
Windows10, Node v14.15.1, Chromium 94.0.4596.0, Cypress 8.7.0

**Reply by AussieFlem** [reply]
We're seeing this too on Win10 and Node 14, latest version of Chrome

**Reply by AussieFlem** [reply]
No proxy set in our case - was working previously so unsure if this is due to changes in 9.4.x

**Comment by jbogheand** [comment]
Ok, so I have finally identified the problem.

Turning off the Network Threat Protection stting in Sophos antivirus solved the problem. Seem like Sophos is fixing it in the next update, but I have not checked that myself.

**Reply by JoeyLi-1** [reply]
Thank you so much for pointing out how to bypass this issue! That is very helpful.

What do you mean by "Seem like Sophos is fixing it in the next update"? Is there any roadmap or development plan for Sophos? I can not find anything on their official website. And our company`s policy does not allow disable that setting. I am really depressed now. @jbogheand

**Reply by jbogheand** [reply]
The responsible for this kind of things in my company has looked it up and told me that Sophos is fixing it, but I have not verified it myself.

**Reply by JoeyLi-1** [reply]
Got it! Thank you!
